packages/camera/camera_windows/pubspec.yaml

```diff
@@ -27,3 +27,9 @@ dev_dependencies:
   async: ^2.5.0
   flutter_test:
     sdk: flutter
+
+
+# FOR TESTING ONLY. DO NOT MERGE.
+dependency_overrides:
+  camera_platform_interface:
+    path: ../../camera/camera_platform_interface
```

Comment:
Don't forget to revert this

Reply:
From my understanding I am supposed to submit this PR without reverting it for now? https://github.com/flutter/flutter/wiki/Contributing-to-Plugins-and-Packages#user-content-changing-federated-plugins

Then create a new one after everything else looks good